src/TodoListItem.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 
-const TodoListItem = ({todo}) => {
+const TodoListItem = ({todo, onRemoveTodo}) => {
     return (
-        <li>{todo.title}</li>
+        <>
+        <li>
+            {todo.title}
+            <button onClick={()=>
+            onRemoveTodo(todo.id)} type="button">Remove</button>
```

Comment:
would make the button a little more readable if you and the onClick and type props/attributes on independent lines: 

<button
    onClick=(() => onRemoveTodo(todo.id)}
    type="button">
    Remove
</button>

(with onclick and type indented one space. the comments aren't letting me do it)